packages/alfa-rules/test/sia-r5/rule.spec.tsx

```diff
@@ -0,0 +1,63 @@
+import { test } from "@siteimprove/alfa-test";
+
+import { Document } from "@siteimprove/alfa-dom";
+
+import R5, { Outcomes } from "../../src/sia-r5/rule";
+
+import { evaluate } from "../common/evaluate";
+import { passed, failed, inapplicable } from "../common/outcome";
+
+test(`evaluate() passes an html element with lang attribute whose value is a valid primary language subtag`, async (t) => {
+    const target = <html lang="en"></html>;
+  
+    const document = Document.of([target]);
+  
+    t.deepEqual(await evaluate(R5, { document }), [
+      passed(R5, target.attribute("lang").get(), {
+        1: Outcomes.HasValidLanguage,
+      }),
+    ]);
+  });
+
+  test(`evaluate() passes an html element with lang attribute whose value is a  valid language tag even though the region subtag is not`, async (t) => {
+    const target = <html lang="en-US-GB"></html>;
+  
+    const document = Document.of([target]);
+  
+    t.deepEqual(await evaluate(R5, { document }), [
+      passed(R5, target.attribute("lang").get(), {
+        1: Outcomes.HasValidLanguage,
+      }),
+    ]);
+  });
+
+  test(`evaluate() fails an html element with lang attribute whose value is not a valid language tag.`, async (t) => {
+    const target = <html lang="em-US"></html>;
```

Comment:
```suggestion
    const target = <html lang="invalid"></html>;
```
Here, the ACT rule was trying to have a "realistic" example (typo `em` instead of `en`). I'd prefer for us to have a "blatantly wrong" example to avoid spending time reading it and trying to understand what is wrong.